hokusai/production.yml

```diff
@@ -24,11 +24,11 @@ spec:
         - name: NODE_ENV
           value: production
         - name: PORT
-          value: 8080
+          value: "8080"
         - name: STATSD_HOST
```

Comment:
i feel like `STATSD_HOST` and `STATSD_PORT` make more sense to be removed from here and added to config maps.
`hokusai production env set STATSD_HOST=datadog STATSD_PORT=8125`

what do you think?

Reply:
I had thought about that. It's appealing to me to keep things in version control, reified in the source, so to speak, unless they're dynamic or sensitive. So, by contrast, the `METRIC_NAME_WHITELIST` I was definitely envisioning being controlled by the config map.

But I'm new to this K8s space. Curious whether there are drawbacks to this?

Reply:
no that makes sense as long as you think its ok to have this env variable checked in into the repo and fine with doing updates via new deployments vs. updating env.

As it is now when you do a `hokusai staging env set ...` you still need to do a `hokusai staging refresh` to bring up new pods with new env variables.